src/CatBox.NET/ICatBox.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace CatBox.NET;
+
+
+/// <summary>
+/// Provides an abstraction over <see cref="CatBoxClient"/> to group multiple tasks together
+/// </summary>
+/// <remarks>Not currently implemented so don't use</remarks>
+public interface ICatBox
+{
+    public Task UploadFile();
+    public Task DeleteFile();
+    public Task CreateAlbum();
+    public Task EditAlbum();
+    public Task AddToAlbum();
+    public Task RemoveFromAlbum();
+    public Task DeleteAlbum();
```

Comment:
~~We don't need `public` here, do we? I've never looked into or used DIMs before. What does that bring to the table here with this?~~

Edit: I guess that was apart of the original implementation for some reason.